src/components/Dashboard/DashboardPieChart.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <el-card class="card">
+    <el-row>
+      <el-col :span="11">
+        <div class="list_crypto">
+          <div class="list_crypto-content">
+            <div v-for="(item, index) in portfolio"
+              :key="index"
+              class="list_crypto-content-item"
+              >
+              <span class="list_crypto-content-color"
+                :style="{ backgroundColor: `#${item.color}` }"></span>
+              <p class="list_crypto-content-asset">{{item.asset}}</p>
+            </div>
+          </div>
+        </div>
+      </el-col>
+      <el-col :span="13">
+        <donat-chart :style="{ margin: '-0.5rem 0rem 0rem 0rem' }" :height="200" :data="portfolio"/>
```

Comment:
I think we should try not to use inline style unless it is being changed actively like a mock.